protobuf/lib/src/protobuf/coded_buffer_writer.dart

```diff
@@ -338,10 +338,10 @@ class CodedBufferWriter {
         break;
       case PbFieldType._BYTES_BIT:
         _writeBytesNoTag(
-            value is TypedData ? value : Uint8List.fromList(value));
+            value is Uint8List ? value : Uint8List.fromList(value));
```

Comment:
I'm amazed that the linter can do this change, but I can't see the implications and whether this is safe (i.e. won't break user code). If I had to guess how the linter was able to do this change I would say it run type flow analysis and figured that the only `TypedData` that flows into this code is `Uint8List`, but I don't really know.

After these changes we could actually update `writeRawBytes` parameter too, but it's a public function (which is probably the reason why the linter didn't update it) and we have internal users so that would be a breaking change.

Reply:
This was a manual change on my part. Trying to get rid of the dynamic invoke.